api/PaymentResponse.json

```diff
@@ -94,22 +94,9 @@
             "chrome": {
               "version_added": "61"
             },
-            "chrome_android": [
-              {
-                "version_added": "56"
-              },
-              {
-                "version_added": "53",
-                "version_removed": "56",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#web-payments",
-                    "value_to_set": "Enabled"
-                  }
-                ]
-              }
-            ],
+            "chrome_android": {
+              "version_added": "53"
```

Comment:
The data now says that PaymentResponse was shipped in Chrome for Android 56, but this subfeature was shipped earlier. That should fail the consistency lint I think, but somehow doesn't. I suspect it's because that lint doesn't deal with flags correctly. IMHO, what it should do is two passes. One that considers only behind-a-flag versions and one that considers only shipping-by-default versions. That should then have failed here.

Reply:
Hmmm...  To be honest, I don't think that this should fail the consistency check, nor should the flag information be repeated for the subfeatures, separating the subfeatures into multiple support statements just to satisfy the rendered tables.  I think this is the same problem as if an API is supported behind a prefix or with an alt. name, when the subfeatures were implemented during those prefixed versions.  (I remember we were dealing with a similar situation in regards to the Geometry APIs.)

Reply:
It looks like there's an open issue about this, see #6509.

Reply:
I just realized that #6670 (irrelevant flag removal) also applies, since the flags have been removed 2+ years ago.  Perhaps we should just remove the flag data for Chrome entirely?

Reply:
Thanks for pointing out https://github.com/mdn/browser-compat-data/issues/6509, I didn't know there were different interpretations of flags and had just assumed everyone shared my own interpretation. Typical 👍 

Once https://github.com/mdn/browser-compat-data/pull/6670 has been merged, I'd love if we can resolve this by just removing the flag data.

For now, can we have the subfeatures represent when things were shipped by default? Even if flag data is taken to be inherited, there would then be no conflict between the parent and child features. (The problematic cases will be large features still entirely behind a flag.)

Reply:
#6670 is merged now, so we can remove the flag data!

Reply:
Oh happy day!